src/main/java/org/betonquest/betonquest/objectives/CommandObjective.java

```diff
@@ -0,0 +1,76 @@
+package org.betonquest.betonquest.objectives;
+
+import org.betonquest.betonquest.BetonQuest;
+import org.betonquest.betonquest.Instruction;
+import org.betonquest.betonquest.api.Objective;
+import org.betonquest.betonquest.exceptions.InstructionParseException;
+import org.betonquest.betonquest.utils.PlayerConverter;
+import org.bukkit.Bukkit;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerCommandPreprocessEvent;
+
+import java.util.List;
+
+/**
+ * Requires the player to execute a specific command.
+ */
+@SuppressWarnings("PMD.CommentRequired")
+public class CommandObjective extends Objective implements Listener {
+
+    private final String command;
+    private final List<String> commandVariables;
+
+    public CommandObjective(final Instruction instruction) throws InstructionParseException {
+        super(instruction);
+        template = ObjectiveData.class;
```

Comment:
```suggestion
```

You dont need this

Reply:
All your other objectives have this specified, I was being consistent. If you want it removed, no problem.

Reply:
some of them are like this and some are like my suggestion. There is a lot of inconsistency: Example: https://github.com/BetonQuest/BetonQuest/blob/master/src/main/java/org/betonquest/betonquest/objectives/PickupObjective.java